benchmarking/prototype_parameterized_template.py

```
@@ -0,0 +1,613 @@
+import hashlib
+import json
+import os
+import platform
+import re
+import statistics
+import subprocess
+import sys
+import tempfile
+import time
+from pathlib import Path
+from typing import Any, Callable
```

Ruff (0.16.0)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)